lib/link_preview/configuration.rb

```diff
@@ -77,5 +79,13 @@ def middleware
     def middleware=(*middleware)
       @middleware = middleware
     end
+
+    def default_content_aspect_ratio
+      @default_content_aspect_ratio ||= (3.0 / 4.0)
```

Comment:
@eatstoomuchjam not sure - I just guessed this from the aspect ratio that YouTube returns. I'll test 16:9 and see if it looks decent